### PR TITLE
DSD-1671: Accessibility update for the focus ring style in the FeedbackBox open button

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-1671](https://jira.nypl.org/browse/DSD-1671)

## This PR does the following:

- Updates the focus ring style for the `FeedbackBox`'s open button to have better contrast against darker backgrounds.
- This specifically fixes the color contrast issue when reviewed in DC Facelift but this applied to other apps.

## How has this been tested?

Added a scrollable section in Storybook to better view the focus ring on various DS background colors.
Screen Shot 2024-01-24 at 2 45 28 PM

## Accessibility concerns or updates

- Fixes the color contrast issue.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/FeedbackBox/FeedbackBox.stories.tsx [hunk 1]
```diff
@@ -54,17 +56,45 @@ const FeedbackBoxWithControls = (args) => {
...
+          </span>
+        }
+        onSubmit={onSubmit}
+        view={internalView}
+        sx={{
+          "> button": {
+            mr: "20px",
+            mb: "20px",
+          },
+        }}
```
Comment (bigfishdesign13) [inline src/components/FeedbackBox/FeedbackBox.stories.tsx]: Did you add this to move the button out so we could see the focus ring around the entire button?  I don't think this is necessary.  Let's just show the true positioning.

File: src/components/FeedbackBox/FeedbackBox.stories.tsx [hunk 2]
```diff
@@ -54,17 +56,45 @@ const FeedbackBoxWithControls = (args) => {
...
+      </Box>
+      <Box height="400px" bgColor="dark.ui.bg.default" width="100%">
+        <Heading level="h3" m="s" color="ui.white">
+          dark.ui.bg.default
+        </Heading>
+      </Box>
+      <Box height="400px" bgColor="dark.ui.bg.page" width="100%">
+        <Heading level="h3" m="s" color="ui.white">
+          dark.ui.bg.page
+        </Heading>
```
Comment (bigfishdesign13) [inline src/components/FeedbackBox/FeedbackBox.stories.tsx:78]: We can change the background color using the feature in Storybook.  But this does work well on the `Docs` page, so let's keep it.  Just add a note that explains what's going on.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
The styles look good, but I have some comments about the docs.

**Review by bigfishdesign13 — APPROVED** [review]
Looks great!

**Comment by EdwinGuzman** [comment]
@avertrees @7emansell just fyi that I'm going to merge this in soon.

**Pull request merged**